api/jwt/jwt.go

```diff
@@ -2,26 +2,31 @@ package jwt
 
 import (
 	"errors"
-
-	"github.com/portainer/portainer/api"
-
 	"fmt"
+	"github.com/portainer/portainer/api"
 	"time"
 
 	"github.com/dgrijalva/jwt-go"
 	"github.com/gorilla/securecookie"
 )
 
+const (
+	// change settings timeout accordingly, current 24h = 86400
+	blacklistTokenTTL = 86400
```

Comment:
Wording/naming:

I suggest `blocklistTokenTTLinSeconds` for two reasons:
- blocklist vs blacklist: complying with more progressive terms speech by using welcoming and inclusive language
- Seconds in order to provide a value unit to avoid interpretation mistakes

Reply:
Done